lte/gateway/deploy/magma_5g_agw/ansible_vars.yaml

```diff
@@ -0,0 +1,19 @@
+# Make it false if you want to use static IP 
+configure_dhcp: true
+
+# Uncomment eth0, eth1, eth0_netmask, eth1_netmask, eth0_gateway, eth1_gateway if you want to use static IP. 
+
+# eth0: ""
+# eth1: ""
+# eth0_netmask: ""
+# eth1_netmask: ""
+# eth0_gateway: ""
+# eth1_gateway: ""
+
+magma_control_proxy_path: "/home/ubuntu/agw-deployment/ansible/roles/magma-agw/files/control_proxy.yaml"
+magma_rootCA_path: "/home/ubuntu/agw-deployment/ansible/roles/magma-agw/files/rootCA.pem"
```

Comment:
Which orc8r this rootCA pointing to. can we not hardcode rootCA or control_proxy